<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>625</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>QPacman</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../libs/qpacman/libqpacman.qrc">
    <normaloff>:/pics/qpacman.svg</normaloff>:/pics/qpacman.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="PacmanView" name="packetView"/>
           </item>
          </layout>
         </widget>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Info</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="PacmanInfoBrowser" name="pacInfoView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Files</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="FilesListWidget" name="filesTree">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true"/>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="PacmanWaitView" name="waitView"/>
       </item>
       <item>
        <widget class="InfoView" name="infoBrowser"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="logWindow"/>
       </item>
       <item>
        <widget class="ProgressView" name="progressView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="actionsToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMark_All"/>
   <addaction name="actionReset"/>
   <addaction name="actionLog"/>
   <addaction name="actionInfo"/>
   <addaction name="actionCancel"/>
   <addaction name="actionApply"/>
  </widget>
  <widget class="PacmanToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefreshList"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionFullUpdate"/>
   <addaction name="actionCacheCleanUp"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionReset">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Resets all changes were done</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionMark_All">
   <property name="icon">
    <iconset theme="show-all-effects">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Mark All</string>
   </property>
   <property name="toolTip">
    <string>Marks all items</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="state-sync">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Updates the repository database and refresh the package list</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="dialog-messages">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>Shows the messages from the packages' post scripts</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionRefreshList">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>RefreshList</string>
   </property>
   <property name="toolTip">
    <string>Refresh package list only</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionCacheCleanUp">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>CacheCleanUp</string>
   </property>
   <property name="toolTip">
    <string>Removes all packages from the pacman's cache</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Sets the selection to the previous item</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Sets the selection to the next item</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionFullUpdate">
   <property name="icon">
    <iconset theme="state-download">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Full Update</string>
   </property>
   <property name="toolTip">
    <string>Full update of the system</string>
   </property>
   <property name="shortcut">
    <string>Alt+U</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="dialog-cancel">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="toolTip">
    <string>Cancel the current operation</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="help-about" resource="../libs/qpacman/libqpacman.qrc">
     <normaloff>:/pics/help-about.svg</normaloff>:/pics/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Show an additional info about this application</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="icon">
    <iconset theme="run-build">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="toolTip">
    <string>Applies the marked actions</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PacmanInfoBrowser</class>
   <extends>QTextBrowser</extends>
   <header>pacmaninfobrowser.h</header>
  </customwidget>
  <customwidget>
   <class>PacmanWaitView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pacmanwaitview.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressView</class>
   <extends>QTreeView</extends>
   <header location="global">progressview.h</header>
  </customwidget>
  <customwidget>
   <class>FilesListWidget</class>
   <extends>QTreeWidget</extends>
   <header>fileslistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PacmanToolBar</class>
   <extends>QToolBar</extends>
   <header>pacmantoolbar.h</header>
  </customwidget>
  <customwidget>
   <class>PacmanView</class>
   <extends>QTreeView</extends>
   <header>pacmanview.h</header>
  </customwidget>
  <customwidget>
   <class>InfoView</class>
   <extends>QTextBrowser</extends>
   <header location="global">infoview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../libs/qpacman/libqpacman.qrc"/>
 </resources>
 <connections/>
</ui>
